src/providers/twitch/TwitchChannel.hpp

```diff
@@ -32,17 +32,11 @@ namespace chatterino {
 // instead.
 // See https://github.com/Chatterino/chatterino2/issues/3384 and
 // https://mm2pl.github.io/emoji_rfc.pdf for more details
-const QString ZERO_WIDTH_JOINER = QString(QChar(0x200D));
+inline const QString ZERO_WIDTH_JOINER = QStringLiteral("\u200D");
```

Comment:
Leave the changes from this file out until we have unit tests added for this RFC